Sources/Features/Client/BacktraceOomWatcher.swift

```diff
@@ -0,0 +1,148 @@
+import Foundation
+
+final class BacktraceOomWatcher {
+    private var applicationState = [String :Any]()
+    
+    
+    private(set) var lowMemoryFilePrefix = "_lowMemory"
```

Comment:
`private(set) var` means that the setter is private but the getter is internal. The rule of thumb: 
- if it never changes and can be defined in the compilation time: `static let` value
- if it never changes but it's a runtime value: `let` value
- if it can change: `private(set) var`

```suggestion
    let lowMemoryFilePrefix = "_lowMemory"
```

Reply:
done, thx